src/platform_impl/linux/x11/mod.rs

```diff
@@ -64,6 +58,11 @@ use crate::{
 const X_TOKEN: Token = Token(0);
 const USER_REDRAW_TOKEN: Token = Token(1);
 
+pub(self) struct WakeSender<T> {
+    pub sender: Sender<T>,
+    pub waker: Arc<Waker>,
+}
```

Comment:
I believe that removing all the `pub` keywords from here would give the same accessibility. (Non-pub fields are accessible to the entire module)

Reply:
You were right, the `pub` keywords aren't necessary. I have removed them.